hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -83,16 +83,16 @@
  * Provides a ledger for Hedera Services crypto and smart contract
  * accounts with transactional semantics. Changes to the ledger are
  * <b>only</b> allowed in the scope of a transaction.
- *
+ * <p>
  * All changes that are made during a transaction are summarized as
  * per-account changesets. These changesets are committed to a
  * wrapped {@link TransactionalLedger}; or dropped entirely in case
  * of a rollback.
- *
+ * <p>
  * The ledger delegates history of each transaction to an injected
  * {@link AccountRecordsHistorian} by invoking its {@code addNewRecords}
  * immediately before the final {@link TransactionalLedger#commit()}.
- *
+ * <p>
```

Comment:
are the added `<p>` tags necessary?

Reply:
i guess it's some IDE setting we can fix